sonic-utilities-tests/config_mgmt_test.py

```diff
@@ -76,6 +76,22 @@ def test_break_out(self):
         self.dpb_port4_4x25G_2x50G_f_l(curConfig)
         return
 
+    def test_extra_tables_cases(self):
+        # make sure no prompt with LOCK
+        curConfig = dict(configDbJson)
+        lock = {"LOCK": {"configDbLock": {"PID": "15555"}}}
+        self.updateConfig(curConfig, lock)
+        cm = self.config_mgmt_dpb(curConfig)
+        # Assert LOCK is removed from Input Config
+        assert len(cm.configdbJsonIn) == 5 and \
```

Comment:
Can we remove this constant and put it as a const variable with class level scope?

Reply:
Not needed, test code, so better keep it upto test function. 
But I think I must use len(curConfig)-1. I will do that change.